### ISSUE TITLE
[REG 1.14.1->1.14.2] Cannot load on Win7 and Ruby 2.5.6

### ISSUE THREAD

**Issue by orgads** [ffi/ffi#868]
```
irb(main):001:0> require 'ffi'
Traceback (most recent call last):
9: from C:/Ruby/bin/irb.cmd:19:in `<main>'
8: from (irb):1
7: from C:/Ruby/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:39:in `require'
6: from C:/Ruby/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:135:in `rescue in require'
5: from C:/Ruby/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:135:in `require'
4: from C:/Ruby/lib/ruby/gems/2.5.0/gems/ffi-1.14.2-x64-mingw32/lib/ffi.rb:3:in `<top (required)>'
3: from C:/Ruby/lib/ruby/gems/2.5.0/gems/ffi-1.14.2-x64-mingw32/lib/ffi.rb:6:in `rescue in <top (required)>'
2: from C:/Ruby/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
1: from C:/Ruby/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
LoadError (cannot load such file -- ffi_c)
```

**Comment by larskanis** [comment]
@orgads I can reproduce your issue. I'll fix this with ffi-1.14.3 in the next few days. Alternatively you can update to RubyInstaller-2.5.8 to fix compatibility.

Background is, that ffi-1.14.2 is built with a newer gcc version, which requires `libssp-0.dll` to run. But the DLL is not present in older RubyInstaller versions. I'll compile libssp into `ffi_c.so` as a static library to fix this.

**Comment by sophieyfang** [comment]
I am using 2.6.5 ruby installer, but still sees the ffi_c loaderror. This happened when the 1.14.2 is released.

Attached error below:
```
PS C:\Users\sophieyfang\Documents> & 'C:\Program Files (x86)\Stackdriver\LoggingAgent\Main\bin\ruby.exe' 'C:\Program Files (x86)\Stackdriver\LoggingAgent\Main\bin\fluentd' -c 'C:\Program Files (x86)\Stackdriver\LoggingAgent\fluent.conf' -x stackdriverlogging -vvv
Traceback (most recent call last):
        16: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/bin/fluentd:23:in `<main>'
        15: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/bin/fluentd:23:in `load'
        14: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/gems/2.6.0/gems/fluentd-1.11.2-x86-mingw32/bin/fluentd:8:in `<top (required)>'
        13: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
        12: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
        11: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/gems/2.6.0/gems/fluentd-1.11.2-x86-mingw32/lib/fluent/command/fluentd.rb:19:in `<top (required)>'
        10: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
         9: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
         8: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/gems/2.6.0/gems/fluentd-1.11.2-x86-mingw32/lib/fluent/supervisor.rb:40:in `<top (required)>'
         7: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
         6: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
         5: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/gems/2.6.0/gems/win32-ipc-0.6.6/lib/win32/ipc.rb:1:in `<top (required)>'
         4: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
         3: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
         2: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/gems/2.6.0/gems/ffi-1.14.2-x86-mingw32/lib/ffi.rb:4:in `<top (required)>'
         1: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require': 126: The specified module could not be found.   - C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/gems/2.6.0/gems/ffi-1.14.2-x86-mingw32/lib/2.6/ffi_c.so (LoadError)
        17: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/bin/fluentd:23:in `<main>'
        16: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/bin/fluentd:23:in `load'
        15: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/gems/2.6.0/gems/fluentd-1.11.2-x86-mingw32/bin/fluentd:8:in `<top (required)>'
        14: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
        13: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
        12: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/gems/2.6.0/gems/fluentd-1.11.2-x86-mingw32/lib/fluent/command/fluentd.rb:19:in `<top (required)>'
        11: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
        10: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
         9: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/gems/2.6.0/gems/fluentd-1.11.2-x86-mingw32/lib/fluent/supervisor.rb:40:in `<top (required)>'
         8: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
         7: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
         6: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/gems/2.6.0/gems/win32-ipc-0.6.6/lib/win32/ipc.rb:1:in `<top (required)>'
         5: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
         4: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
         3: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/gems/2.6.0/gems/ffi-1.14.2-x86-mingw32/lib/ffi.rb:3:in `<top (required)>'
         2: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/gems/2.6.0/gems/ffi-1.14.2-x86-mingw32/lib/ffi.rb:6:in `rescue in <top (required)>'
         1: from C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
C:/Program Files (x86)/Stackdriver/LoggingAgent/Main/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require': cannot load such file -- ffi_c (LoadError)
```

**Comment by larskanis** [comment]
This issue should be fixed with ffi-1.15.0
